src/routes/homepage-redesign-preview/index.svelte

```diff
@@ -1,106 +1,260 @@
   import { Card, CardsContainer } from "~/components/cards";
+  import { Banner, SidebarRight } from "~/partials";
+  import { InlineNotification } from "carbon-components-svelte";
+  import InformationFilled32 from "carbon-icons-svelte/lib/InformationFilled32";
 
-  const dummyLink = "/homepage-redesign-preview/#";
-
-  const cardsData = [
-    {
-      titleText: "Tools, How to Guide, & Data Download",
-      linkPath: "/",
-      description: `Access climate data via visualizations, downloads, and
-        the Cal-Adapt API for your research, outreach, and adaptation planning
-        needs.`,
-      bgColor:
-        "linear-gradient(180deg, #0F9699 0%, #0F9982 0.01%, #035466 100%)",
-      ctaText: "Learn More",
-    },
-    {
-      titleText: "Cal-Adapt: Analytics Engine",
-      linkPath: "https://analytics.cal-adapt.org",
-      description: `Utilize cloud storage and computation to analyze Petabytes
-        of stored data supported by scientific guidance.`,
-      bgColor: "linear-gradient(180deg, #248DB3 0%, #096180 100%)",
-      ctaText: "Visit Site",
-    },
-    {
-      titleText: "Data Development Grants",
-      linkPath: dummyLink,
-      description: "Description forthcoming.",
-      bgColor: "linear-gradient(180deg, #19953E 0%, #196B5B 100%)",
-      ctaText: "Learn More",
-    },
-  ];
-  const cardHeight = 20;
-  const cardWidth = 18;
+  export let events;
+  export let posts;
+  export let cardsData;
+  export let revealNewHomepage;
 
-  const updateLinksPlaceholders = [
-    {
-      title: "Update number one",
-      linkPath: dummyLink,
-    },
-    {
-      title: "Update number two",
-      linkPath: dummyLink,
-    },
-    {
-      title: "Update number three",
-      linkPath: dummyLink,
-    },
-  ];
+  const icons = [
+    "sun",
+    "rainfall",
+    "wildfire",
+    "snowflake",
+    "sea",
+    "streamflow",
+  ].map((name) => `img/icons/${name}.svg`);
+  const cardHeight = 18;
+  const cardWidth = 18;
 </script>
 
+<style lang="scss">
+  @import "scss/site/mixins/media-queries";
+
+  .alert :global(.bx--inline-notification) {
+    margin: 0;
+    max-width: 100%;
+    background: $teal-10;
+
+    &::before {
+      border: none;
+    }
+  }
+
+  .alert :global(.bx--inline-notification__text-wrapper) {
+    flex-direction: column;
+  }
+
+  .bx--inline-notification__subtitle {
+    margin-bottom: 0;
+  }
+
+  .btn-container {
+    margin-top: 2.5rem;
+    text-transform: uppercase;
+  }
+
+  .ae-logo-box {
+    padding: 14px;
+
+    img {
+      display: block;
+      margin: auto;
+    }
+  }
+
+  .ae-logo-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    align-items: flex-start;
+
+    @include media("<=medium") {
+      padding-top: 24px;
+    }
+  }
+
+  .cta-card {
+    width: 350px;
+
+    @include media(">large") {
+      float: right;
+    }
+
+    @include media("<=medium") {
+      margin-top: 24px;
+    }
+  }
+
+  .icon-circle {
```

Comment:
Remove `icon-cirlce` selector. This css selector is unused and throws a warning while building the project.